lib/dotcom_web/templates/trip_plan/_leg_summary.html.heex

```diff
@@ -8,6 +8,9 @@
     <%= if @itinerary_row.trip do %>
       <div class="fs-14">
         <%= Routes.Route.direction_name(@route, @itinerary_row.trip.direction_id) %>
+        <%= if @itinerary_row.trip.name && @itinerary_row.trip.name != "" do %>
```

Comment:
This'll output "Outbound 753 towards Headsign", but I'm interpreting the ticket acceptance criteria as wanting "Train 753 outbound towards Headsign"

> Trip plans with a commuter rail leg should be in format of "Train "number" outbound towards Newburyport" 
Ex. for first screenshot below, would say "Train 23 outbound towards Newburyport"

Reply:
I fail at reading.